eva-accession-clustering/src/test/java/uk/ac/ebi/eva/accession/clustering/batch/io/clustering_writer/IssueAccessionClusteringWriterTest.java

```diff
@@ -150,7 +164,7 @@ public void tearDown() {
     @DirtiesContext
     public void writer() throws Exception {
         List<SubmittedVariantEntity> submittedVariantEntities = createSubmittedVariantEntities();
-        clusteringWriter.write(submittedVariantEntities);
+        this.processMergesAndSplits(submittedVariantEntities);
```

Comment:
Due to re-strategization of the clustering process to use a staged approach, clustering writer alone won't cut it because it only generates merge/split candidates. Subsequent merge and split processing also have to be carried out.